build/control

```diff
@@ -1,5 +1,5 @@
 Package: dgtcentaurmods
-Version: 0
+Version: 0.0.1
```

Comment:
This gets changed by build.sh anyway depending on repo tag. Is there a reason for 0.0.1? Wobt affect sed anyway, so I think it looks better anyway :)